### ISSUE TITLE
Suggestion: Replace use of "title" attribute with "data-title"

### ISSUE THREAD

**Issue by townxelliot** [01org/appframework#333]
Rather than setting a "title" attribute on a panel, which is then used to populate the header, use a "data-title" attribute instead. (The use of "title" could be maintained in parallel to prevent API breakage.)

The main issue with "title" is that this is converted by browsers into a tooltip when you hover over a panel, which is at best annoying.

I would be willing to supply a patch, but I'm not sure where I would need to change stuff. My guess is in ui/src/appframework.ui.js, but this seems to have a lot less code in it than ui/appframework.js. What is the relationship between these files and how do I go about a build/test/lint/documentation cycle? (NB even if you're not interested in changing how title attributes are used, it would be good to see some docs on how to contribute to the project :) I could raise that as a separate issue unless I've missed some obvious docs which are already available.)

Thanks.
